godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/type/Transform2D.kt

```diff
@@ -192,7 +192,9 @@ class Transform2D(var x: Vector2, var y: Vector2, var origin: Vector2) : CoreTyp
      * Returns true if this transform and transform are approximately equal, by calling is_equal_approx on each component.
      */
     fun isEqualApprox(transform: Transform2D): Boolean {
-        return transform.x.isEqualApprox(this.x) && transform.y.isEqualApprox(this.y) && transform.origin.isEqualApprox(this.origin)
+        return transform.x.isEqualApprox(this.x) && transform.y.isEqualApprox(this.y) && transform.origin.isEqualApprox(
+            this.origin
+        )
```

Comment:
Really minor detail but since everything else is pretty perfectly formatted, i would format this as well so the `&&` are on a new line or something similar

Reply:
It's supposed to be formatted as well. Maybe the linter only use new lines where there is more than 3 arguments.

Reply:
Yeah the linter only breaks on certain lenghts and then only to the next "breakable" point. Lines like that mostly need manual formatting